### DISCUSSION TITLE
How can i go to right definition in cypress

### DISCUSSION THREAD

**Discussion by urayoru113** [typescript-language-server/typescript-language-server#858] [General]
hello, i use neovim

In cypress

![image](https://github.com/typescript-language-server/typescript-language-server/assets/22545315/c74bec1f-1fab-4f1c-bec0-f744eb0e2052)

and I do `:lua vim.lsp.buf.definition()<CR>`

then I jump to `command.d.ts`

![image](https://github.com/typescript-language-server/typescript-language-server/assets/22545315/9a09d193-caff-4705-8e6c-593ef184390e)

how can i jump to `command.js`

![image](https://github.com/typescript-language-server/typescript-language-server/assets/22545315/1e912dc8-9b3a-4a74-9c27-7a47d667920c)

**Comment by rchl** [comment]
There is a custom command for that: https://github.com/typescript-language-server/typescript-language-server#go-to-source-definition

Ideally there would also be an option to use that by default but there is none yet.
